Documentation/Diagnostics/PH2054.md

```diff
@@ -0,0 +1,23 @@
+# PH2054: 
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MoqAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MoqAnalyzers) |
+| Diagnostic ID | PH2054 |
+| Category  | [Runtime Failure](../RuntimeFailure.md) |
+| Analyzer | [MockRaiseArgumentsMustMatchEventAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MoqAnalyzers/MockRaiseArgumentsMustMatchEventAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
+| Enabled By Default | Yes |
+
+## Introduction
+
+`Mock<T>.Raise(x => x.Event += null, sender, args)` must have correct parameters. There is an argument code mismatch.
```

Comment:
Duplicate of PH2053 ??

Reply:
Change "code" to "count".  this is count mismatch. 2053 is type mismatch.